frontend/src/features/permits/components/dashboard/tests/integration/fixtures/getActiveApplication.ts

```diff
@@ -190,7 +190,7 @@ export const getDefaultApplication = () => {
       expiryDate,
       contactDetails,
       vehicleDetails,
-      commodities,
+      commodities: conditions,
```

Comment:
is this commodities or conditions? do they mean the same thing?

Reply:
"commodities" have to be used as the key inside the permitData json (and everywhere that uses permit data), but "conditions" is what it's actually supposed to mean, and in this case it just the name of the variable assigned to the "commodities" key inside permitData

Reply:
Hmmm I see. What will be `commodities` called going forward? Is this documented somewhere as this can cause confusion once we lose context?

Reply:
It's still going to be named "commodities" to ensure backward compatibility, and the actual commodities will be named "permitCommodities", according to Praveen and John, who decided that this will be done for the permitData json. I'm not sure where this is documented, but I think Praveen/John is working on this data structure right now, so they might have an idea?

Reply:
Actual commodity is 'permittedCommodity' as per the sample DS shared by John